packages/abi-gen/package.json

```diff
@@ -11,38 +11,37 @@
         "lint": "tslint --format stylish --project . && yarn lint-contracts",
         "lint-contracts": "solhint -c ../../contracts/.solhint.json test-cli/contracts/*.sol",
         "fix": "tslint --fix --format stylish --project . && yarn lint-contracts",
-        "clean": "shx rm -rf lib test-cli/**/generated-wrappers",
-        "build": "tsc -b",
-        "test_cli:clean": "yarn clean:sol && rm -rf test-cli/**/generated-wrappers && rm -rf test-cli/typescript/lib",
-        "test_cli:prebuild": "yarn compile:sol && yarn generate_contract_wrappers",
-        "test_cli:build": "tsc --project test-cli/tsconfig.json",
-        "test_cli": "yarn test_cli:run_mocha && yarn diff_contract_wrappers",
-        "build:ci": "yarn build && yarn test_cli:prebuild && yarn test_cli:build",
-        "test": "yarn run_mocha",
-        "run_mocha": "mocha --require source-map-support/register --require make-promises-safe lib/test/*_test.js --timeout 100000 --bail --exit",
-        "test_cli:run_mocha": "mocha --require source-map-support/register --require make-promises-safe test-cli/typescript/lib/**/*_test.js --timeout 100000 --bail --exit",
+        "clean": "shx rm -rf lib test-cli/**/output",
```

Comment:
add in `&& yarn test_cli:clean`.  didn't realize it before, but if we integrated with `yarn build` and then we should integrate with `yarn clean` too.